src/utils/convert.rs

```diff
@@ -69,6 +81,31 @@ impl TryConvertFrom<[i32]> for AVec<AVec<u64>> {
     }
 }
 
+impl<'a> TryConvertFromIter<'a, f64> for Vec<Vec<u64>> {
+    type Parameters = [u64];
+    // TODO we have to transpose the matrix, becasue we can only use the iter once.
+    fn try_convert_from(
+        iter: impl Iterator<Item = &'a f64>,
+        parameters: &Self::Parameters,
+    ) -> Self {
+        iter.map(|signed_x| {
+            parameters
+                .iter()
+                .map(|qi| {
+                    let v = signed_x.abs() as u64;
```

Comment:
f64 may not fit in u64. 

A better approach will be to convert f64 to BigUint (mod Q) and reduce `xi mod Q` to u64 (i.e. by primes in Q's moduli chain [q0, q1, ...]). 

Also since we recently switched default float from f64 to BigFloat (check [here](https://github.com/gaussian-dev/gauss/blob/2a1cb7e6fc7fdcd3d7b3eeeff7dff91a4222715e/src/schemes/ckks/default_impl/entities.rs#L31)). We should also implement the same trait for BigFloat.